external/upstream/libint2/CMakeLists.txt

```diff
@@ -24,17 +25,16 @@
 # * If ever use a libint2 with AM>7, edit the `init_spherical_harmonics` line in libmints/integral.cc
 # * In selecting AM values, before `grep`ing lots of basis set files, consult the guide at end of this file.
 
-
 # CHOOSE! `make export`
 #find_package(Libint2 CONFIG)
 # CHOOSE! pure cmake
-find_package(Libint2 CONFIG COMPONENTS gss "e${MAX_AM_ERI}" g2 h2 eri3_e4 eri3_g3 eri3_h3 eri2_e4 eri2_g3 eri2_h3)
+find_package(Libint2 CONFIG COMPONENTS gss CXX_ho impure_sh "eri_c4_d0_l${MAX_AM_ERI}" eri_c4_d1_l2 eri_c4_d2_l2 eri_c3_d0_l4 eri_c3_d1_l3 eri_c3_d2_l3 eri_c2_d0_l4 eri_c2_d1_l3 eri_c2_d2_l3 onebody_d0_l4 onebody_d1_l3 onebody_d2_l3)
```

Comment:
:scream:

Reply:
yeah, it's a little verbose, but that's a pretty minimal build AM-wise. it's going to be interesting to find a set of build conditions that all the OS QC packages can be content with.